pkgs/top-level/all-packages.nix

```diff
@@ -36797,6 +36797,8 @@ with pkgs;
 
   ivan = callPackage ../games/ivan { };
 
+  ironwail = callPackage ../games/ironwail { };
```

Comment:
While I'm perfectly aware that `all-packages.nix` is not sorted perfectly, I would kindly ask you to avoid adding entropy to this file and move this entry before `ivan` and after `iortcw_sp`.